src/osp/Active/SysVehicle.cpp

```diff
@@ -439,26 +432,33 @@ void SysVehicle::update_vehicle_modification(ActiveScene& rScene)
 
             islands[0] = vehicleEnt;
 
+            Vector3 const velocity = rReg.get<ACompPhysLinearVel>(vehicleEnt);
+
             // NOTE: vehicleVehicle and vehicleTransform become invalid
             //       when emplacing new ones.
             for (size_t i = 1; i < islands.size(); i ++)
```

Comment:
change this to

for(auto & currIsland : islands)
{

....

currIsland = islandEnt;
}

Reply:
Going to use `for (ActiveEnt& rIslands : Corrade::Containers::arrayView(islands).suffix(1))` instead. Note that `i` is initialized to 1, skipping the first element.

Reply:
:+1: